### PR TITLE
Converting the refunds and credits KB to devdocs

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) migrates the https://support.magento.com/hc/en-us/articles/115003067873-Work-with-refunds-and-credit-memos-using-Magento-API topic to devdocs.

## Affected DevDocs pages

- http://devdocs.magento.com/guides/v2.2/rest/modules/sales/refunds.html

## Links to Magento source code

whatsnew
Added information about how to use the REST API to [issue refunds](http://devdocs.magento.com/guides/v2.2/rest/modules/sales/refunds.html).

### CODE REVIEW

File: guides/v2.2/rest/modules/sales/refunds.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+With this service, you can initiate and process a refund based on an Invoice ID, created using an online payment method.
+
+### Endpoint
+
+`POST V1/invoice/:invoiceId/refund`
+
+{:.bs-callout-warning}
+If you try to apply the service to an invoice created using an offline payment method, the system will display a validation error.
+
+The `salesRefundInvoice` service allows you to:
```
Comment (keharper) [inline guides/v2.2/rest/modules/sales/refunds.md]: Move this section to the intro of the H2.

File: guides/v2.2/rest/modules/sales/refunds.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+* Create a credit memo (complete or partial) for an invoice
+* Add details about the refunded items to the order
+* Change status and state of an order according to performed actions
+* Notify a customer about the performed refund operation
+* Designate whether the returned items are returned to stock
+
+## salesRefundOrderV1 service
+
+This service performs the same operations as the `salesRefundInvoiceV1` service, but based on an Order ID. Use this service if the invoice was created using an offline payment method.
```
Comment (keharper) [inline guides/v2.2/rest/modules/sales/refunds.md]:
```suggestion
This service performs the same operations as the `salesRefundInvoiceV1` service, but based on an order ID. Use this service if the invoice was created using an offline payment method.
```

File: guides/v2.2/rest/modules/sales/refunds.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+### Endpoint
+
+`POST V1/order/:orderId/refund`
```
Comment (keharper) [inline guides/v2.2/rest/modules/sales/refunds.md]: Add the same warning as above, except change "offline" to "online".

File: guides/v2.2/rest/modules/sales/refunds.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## Related documentation
+
+* [Magento REST API documentation (Swagger)](https://devdocs.magento.com/swagger/index.html)
```
Comment (keharper) [inline guides/v2.2/rest/modules/sales/refunds.md]: Delete this link. 

Change the heading to something like Other services for issuing refunds

File: guides/v2.2/rest/modules/sales/refunds.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+### Endpoint
+
+`POST V1/order/:orderId/refund`
+
+{:.bs-callout-warning}
+If you try to apply the service to an invoice created using an online payment method, the system will return a validation error.
+
+## Other services for issuing refunds
+
+While Magento recommends you use the `Refund` services, the Magento API also provides the following `CreditmemoManagement` services you can use to issue a credit:
```
Comment (keharper) [inline guides/v2.2/rest/modules/sales/refunds.md]:
```suggestion
While Magento recommends you use the `Refund` services, Magento also provides the following `CreditmemoManagement` services you can use to issue a credit:
```

**Comment by erikmarr** [comment]
running tests

**Pull request merged**